run-tests.php

```diff
@@ -2123,7 +2123,11 @@ function run_test($php, $file, $env)
 				@unlink($test_file);
 			}
 
-			if (!$leaked && !$failed_headers) {
+			// check if EXPECT isn't better
+			if (isset($section_test['EXPECTF']) && preg_match('/%a|%e|%i|%u|%b|%c|%d|%e|%f|%r|%s|%w|%x/i', $wanted) === 0) {
```

Comment:
Typo here: `$section_test` instead of `$section_text`.

Reply:
Should I fixed it and rePR?